libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -833,6 +833,18 @@ bool ASTJsonConverter::visit(Literal const& _node)
 	return false;
 }
 
+bool ASTJsonConverter::visit(FormalDocumentation const& _node)
+{
+	Json::Value text{*_node.text()};
+	std::vector<pair<string, Json::Value>> attributes = {
+		make_pair("text", text),
+		make_pair("format", "NatSpec")
```

Comment:
This indicates that the NatSpec format used to parse documentation in source. Since it's the only available format right now, it's hard-coded here and in the AST JSON importer. Could be replaced by an enum in `FormalDocumentation` in the future if needed.

Reply:
My issue is that this is not really natspec. This is just doxygen / javadoc. Natspec is the formatting inside of `@userdoc`.

Reply:
Clarified in the chat. I'll remove the attribute then. Thanks.